src/RadixTreeUnOrdered.elm

```diff
@@ -0,0 +1,80 @@
+module RadixTreeUnOrdered exposing
+    ( RadixTreeUnOrdered
+    , empty, singleton, insert, toTree
+    )
+
+{-| Build an ordered Radix tree.
+
+
+# Structure
+
+@docs RadixTreeUnOrdered
+
+
+# Build
+
+@docs empty, singleton, insert, toTree
```

Comment:
`toTree` is not a function to "build" a tree

Reply:
Moved under `Structure`